unit/solvers/smt2_incremental/smt2_incremental_decision_procedure.cpp

```diff
@@ -464,6 +466,10 @@ TEST_CASE(
   const auto invalid_pointer_object_size_definition =
     test.object_size_function.make_definition(
       1, smt_bit_vector_constant_termt{0, 32});
+  const auto null_object_dynamic_definition =
+    test.is_dynamic_object_function.make_definition(0, false);
```

Comment:
❓ Where do we handle the `null_object_dynamic_definition` case?

Reply:
It is currently defaulted to false in the definition of `decision_procedure_objectt`. I should set it explicitly in the `make_null_object` function.

Reply:
Done.